release-schema.json

```
@@ -198,6 +290,32 @@
           "type": "string",
           "description": "A local identifier for this group of lots."
         },
+        "identifiers": {
+          "title": "Identifiers",
+          "description": "Identifiers for this lot group. This field can be used to provide internal identifiers for the lot group, such as identifiers from a buyer's document management system or procurement planning system.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SimpleIdentifier"
+          },
+          "uniqueItems": true,
+          "wholeListMerge": true
+        },
+        "title": {
+          "title": "Title",
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "A title for this lot group."
```

description after title.

```suggestion
          "description": "A title for this lot group."
          "type": [
            "string",
            "null"
          ],
```